source/plugin/effects.rst

```diff
@@ -63,24 +63,29 @@ Similarly to sounds, we can use the ``Viewer`` class to spawn particles within t
     import org.spongepowered.api.effect.particle.ParticleTypes;
     
     ParticleEffect effect = ParticleEffect.builder()
-        .type(ParticleTypes.LAVA).count(50).build();
+            .type(ParticleTypes.LAVA)
+            .quantity(50)
+            .build();
     viewer.spawnParticles(effect, position);
 
 Using a :javadoc:`ParticleEffect.Builder`, we can specify the type of particle we
 would like to spawn. With this, we also specify that fifty particles will be in the particle effect.
 
 Now if we wanted to make a more specific particle, say the particle of a block, then we can use one of the serveral
-classes found in the :javadoc:`org.spongepowered.api.effect.particle` package. For example, let's say we wanted to
-spawn the particle of a sand, :javadoc:`ParticleTypes#BLOCK_CRACK`. We would need to use the :javadoc:`BlockParticle`
-class and specify that we would like to use a sand block. This can be done like so:
+types found in the :javadoc:`ParticleTypes` class. For example, let's say we wanted to spawn the particle of a sand,
```

Comment:
Do you mean "spawn the particle of sand" or "spawn the particle of a sand block"? This might work better using the plural too, ie. "spawn particles of ...".